apps/teams-test-app/e2e-test-data/mail.handoff.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "Handoff ComposeMail",
+  "platforms": "*",
+  "featureTests": [
+    {
+      "feature": {
+        "id": "mail.handoff",
+        "version": 1
+      },
+      "testCases": [
+        {
+          "title": "checkMailWithHandoffCapability API Call - Success",
+          "type": "callResponse",
+          "boxSelector": "#box_checkMailWithHandoffCapability",
+          "expectedTestAppValue": "MailWithHandoff module is supported"
+        },
+        {
+          "title": "mail.handoff.composeMail Feature Test - Success",
+          "type": "callResponse",
+          "boxSelector": "#box_composeMailWithHandoff",
+          "inputValue": {
+            "composeMailParams": {
+              "type": "new",
+              "subject": "Test Mail",
+              "toRecipients": ["sam@example.com", "sam1@example.com"],
+              "ccRecipients": ["sam2@example.com"],
+              "bccRecipients": ["sam3@example.com"],
+              "message": "This mail has been sent from the Teams Test App"
+            },
+            "handoffId": "abc123"
+          },
+          "expectedAlertValue": "composeMailParamsWithHandoff called with {\"type\":\"new\",\"subject\":\"Test Mail\",\"toRecipients\":[\"sam@example.com\",\"sam1@example.com\"],\"ccRecipients\":[\"sam2@example.com\"],\"bccRecipients\":[\"sam3@example.com\"],\"message\":\"This mail has been sent from the Teams Test App\"} and handoffId: abc123",
+          "expectedTestAppValue": "Completed"
+        },
+        {
+          "title": "mail.handoff.composeMail Feature Test with invalid handoffId- Failure",
+          "type": "callResponse",
+          "boxSelector": "#box_composeMailWithHandoff",
+          "inputValue": {
+            "composeMailParams": {
+              "type": "new",
+              "subject": "Test Mail",
+              "toRecipients": ["sam@example.com", "sam1@example.com"],
+              "ccRecipients": ["sam2@example.com"],
+              "bccRecipients": ["sam3@example.com"],
+              "message": "This mail has been sent from the Teams Test App"
+            },
+            "handoffId": ""
+          },
+          "expectedTestAppValue": "Error: Error: handoffId should not be null or empty string."
```

Comment:
I think "Error: Error: ..." bug has been fixed? @TrevorJoelHarris , don't we?

Reply:
As far as I know this still shows up in the Test App :(

Reply:
(And it's sort of hard to fix because of how many tests have been written assuming it's there)

Reply:
Okay, I will leave them to be existed still... I thought we have fixed them.